src/extensions/default/bramble/lib/MouseManagerRemote.js

```diff
@@ -1,6 +1,21 @@
 /*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, forin: true, maxerr: 50, regexp: true, bitwise: true */
 /* global addEventListener, removeEventListener, sessionStorage */
-(function(transport) {
+
+var testSessionStorage = "testSessionStorage";
+var noop = function() {};
+var sessionStorage = {
+  getItem: noop,
+  setItem: noop
+};
+
+try {
+  window.sessionStorage.setItem(testSessionStorage, testSessionStorage);
+  window.sessionStorage.removeItem(testSessionStorage);
+  sessionStorage = window.sessionStorage.bind(window);
+} catch(e) {
+  console.warn("[Bramble] Session storage not accessible for MouseManager.");
+
```

Comment:
You are missing a closing `}`.  It looks like you've never tested this code, since this would never work.  You need to make sure your code a) works; b) passes linting before you submit a PR.